device-types/Ciena/5171-platform.yaml

```diff
@@ -0,0 +1,145 @@
+---
+manufacturer: Ciena
+model: 5171 Platform
+slug: ciena-5171-platform
+part_number: 170-5171-910
+u_height: 2
+weight: 15.5
+weight_unit: kg
+is_full_depth: false
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it

Reply:
removed